src/components/form/index.tsx

```diff
@@ -76,37 +70,54 @@ export class SmoothlyForm implements Clearable, Submittable, Editable {
 		event.detail(this.readonly)
 	}
 	@Method()
-	async submit(): Promise<void> {
-		this.smoothlyFormSubmit.emit(this.value)
+	async submit(remove?: boolean): Promise<void> {
+		this.processing = true
+		this.smoothlyFormSubmit.emit({ value: this.value, type: remove == true ? "remove" : this.type })
 		if (this.action) {
 			const action = this.action
 			this.notice?.emit(
 				Notice.execute("Submitting form", async () => {
 					const response = await http
 						.fetch(
-							this.method == "POST"
-								? http.Request.create({
-										method: "POST",
-										url: action,
-										header: { contentType: "application/json" },
-										body: this.value,
-								  })
-								: { url: `${action}?${http.Search.stringify(this.value)}` }
+							http.Request.create(
+								remove == true
+									? { method: "DELETE", url: action, header: { contentType: "application/json" }, body: this.value }
+									: this.type == "fetch"
+									? {
+											method: "GET",
+											url: `${action}?${http.Search.stringify(this.value)}`,
+									  }
+									: this.type == "change"
```

Comment:
both branches here results in the same object but with a different method. merge these two branches into a single else branch that only changes the method.